src/solvers/qbf/qbf_bdd_core.cpp

```diff
@@ -326,17 +339,19 @@ void qbf_bdd_coret::lcnf(const bvt &bv)
 
   BDD clause(bdd_manager->bddZero());
 
+  // Ranged for
```

Comment:
That'll teach me for not thoroughly checking this after starting it before Christmas and then leaving for 2 weeks!